data_processors/pipeline/lambdas/tests/test_dragen_tso_ctdna.py

```diff
@@ -212,7 +212,7 @@ def test_sqs_handler(self):
         logger.info("Example dragen_tso_ctdna.sqs_handler lambda output:")
         logger.info(json.dumps(results))
 
-        self.assertEqual(len(results), 1)
+        self.assertEqual(len(results), 2)
```

Comment:
Could add a note why we are expecting 2 results, i.e. one for the handler `results` and one for the `batchItemFailures`. No matter what, both should always be present, but may be empty lists.

Reply:
That's the quiz! j/k 😬 Okie dokie, I will add comment in.

Reply:
Ah, you are just checking if I pay attention... fair enough!

Reply:
⏫ https://github.com/umccr/data-portal-apis/commit/09c4975a447a7590a8ecde135095a371119323b5